docs/docs/usage/project.md

```diff
@@ -247,6 +247,20 @@ pdm export -f setuppy -o setup.py
 !!! NOTE
     You can also run `pdm export` with a [`.pre-commit` hook](advanced.md#hooks-for-pre-commit).
 
+## Working with version control
+
+Ensure the `pdm.lock` and `pyproject.toml` files are committed when using a version control system with PDM. Do not commit the `.pdm.toml` file.
```

Comment:
`pyprojec.toml` is a MUST, while `pdm.lock` is a SHOULD. There are practices you don't commit the `pdm.lock` file, especially for a library. `.pdm.toml` is a MUST NOT.

Reply:
Awesome, Ya that makes sense was not aware of a scenario where you would not want to commit the lock file. 

Have updated accordingly. Had to do some rewording for it to make sense but lemme know if any more concerns.